addons/l10n_be_invoice_bba/migrations/12.0.1.2/post-migration.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+from psycopg2.extensions import AsIs
+
+
+def map_res_company_invoice_reference_type(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE res_company
+        SET invoice_reference_type = 'structured'
+        WHERE invoice_reference_type = 'bba'"""
+    )
+
+
+def map_res_company_l10n_be_structured_comm(cr):
```

Comment:
I am not sure of this

Reply:
Discarded because https://github.com/OCA/OpenUpgrade/pull/1941 was also discarded. Too narrow cases, not worth.